startdwarf.lua

```diff
@@ -1,17 +1,79 @@
--- change number of dwarves on initial embark
+--@ module=true
```

Comment:
The first line should be kept for `ls` output, I think.

Reply:
This is only true if there is no accompanying .rst doc file. In-tree scripts don't need any in-script docs.